code/datums/mood_events/generic_negative_events.dm

```diff
@@ -212,6 +212,14 @@
 	mood_change = -8
 	timeout = 3 MINUTES
 
+/datum/mood_event/deaths_door
+	description = "<span class='boldwarning'>This is it... I'm really going to die.</span>\n"
+	mood_change = -20
+
+/datum/mood_event/gunpoint
+	description = "<span class='boldwarning'>This guy is insane! I better be careful....</span>\n"
```

Comment:
police officer: *points a disabler at you and asks you kindly to come quietly*
gunpoint moodlet: "This guy is INSANE!"
maybe make the gunpoint moodlet only trigger when someone points a LETHAL weapon at you?